src/App.jsx

```diff
@@ -1,54 +1,27 @@
+import React from 'react';
 import './App.scss';
 
 import todosFromServer from './api/todos.json';
 import usersFromServer from './api/users.json';
 
-function getUserById(userId) {
-  return usersFromServer.find(user => user.id === userId)
-      || null;
+import { TodoList } from './components/TodoList';
+
+function getUser(userId) {
+  const foundUser = usersFromServer.find(user => user.id === userId);
```

Comment:
```suggestion
  const foundUser = usersFromServer.find(user => user.id === userId) || null;
```
It would be better to return null in case user not found